src/sass/main.scss

```
@@ -1,3 +1,50 @@
-body {
-    background-color: white;
+$_base-space: 1rem * 0.25;
+$space-1: $_base-space * 1;
+$space-2: $_base-space * 2;
+
+// Breakpoint note
+// mobile: 0px ~ 768px
+// tablet: 769px ~ 1024px
+// desktop: 1025px ~
+$break-point-mobile-tablet: 769px;
+$break-point-tablet-desktop: 1025px;
+
+$container-max-width: 1440px;
+
+@mixin mobile {
+  @media screen and (max-width: $break-point-mobile-tablet) {
+    @content;
+  }
+}
+
+@mixin tablet {
+  @media screen and (min-width: $break-point-mobile-tablet) and (max-width: $break-point-tablet-desktop) {
+    @content;
+  }
+}
+
+@mixin desktop {
+  @media screen and (min-width: $break-point-tablet-desktop) {
+    @content;
+  }
+}
```

```suggestion
@mixin desktop {
  @media screen and (min-width: $break-point-tablet-desktop) {
    @content;
  }
}

@mixin tablet {
  @media screen and (min-width: $break-point-mobile-tablet) and (max-width: $break-point-tablet-desktop) {
    @content;
  }
}

@mixin mobile {
  @media screen and (max-width: $break-point-mobile-tablet) {
    @content;
  }
}
```

It might be better to make order consistent. Currently the order of container part is, desktop -> tablet -> mobile but mixin that of mixin part is opposite.

---

:+1:  Thank you